Game/src/Menu.java

```diff
@@ -1,121 +1,284 @@
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.scene.control.Button;
 import javafx.scene.control.Label;
+import javafx.scene.effect.DropShadow;
 import javafx.scene.image.Image;
 import javafx.scene.image.ImageView;
+import javafx.scene.input.KeyCode;
 import javafx.scene.layout.Pane;
 import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
 import javafx.scene.paint.Color;
-import javafx.scene.text.Font;
-import javafx.scene.text.Text;
+import javafx.scene.shape.Rectangle;
+
+import java.io.IOException;
 
 public class Menu {
 
-    public static Scene menu(){
+    @SuppressWarnings("Duplicates")
+    public static Scene mainMenu() {
 
         StackPane root = new StackPane();
         Scene scene = new Scene(root, MainApplication.WINDOW_SIZE_X, MainApplication.WINDOW_SIZE_Y);
 
-        scene.getStylesheets().add("/assets/ui/style.css");
+        scene.getStylesheets().add("/assets/ui/stylesheets/style.css");
+
+        ImageView ivBG = new ImageView(new Image("/assets/ui/backgrounds/alt_main_menu_bg_placeholder.png"));
 
         Button btnStart = new Button();
         Button btnHelp = new Button();
         Button btnExit = new Button();
-        ImageView title = new ImageView(new Image("/assets/ui/pTitleArt.png"));
 
         btnStart.setText("Start");
+        btnStart.setTranslateY(-80);
+        btnStart.setOnAction(e -> MainApplication.getStage().setScene(MainApplication.getGameScene(MainApplication.levelQueue.peek())));
+
         btnHelp.setText("Help");
-        btnExit.setText("Exit");
+        btnHelp.setTranslateY(0);
+        btnHelp.setOnAction(e -> MainApplication.getStage().setScene(helpMenu(scene)));
 
-        btnStart.setTranslateY(-80);
+        btnExit.setText("Exit");
         btnExit.setTranslateY(80);
+        btnExit.setOnAction(e -> MainApplication.exit());
 
-        btnStart.setOnAction(e ->
-            MainApplication.getStage().setScene(MainApplication.getGameScene("./Game/src/assets/levels/level-1"))
-        );
+        root.getChildren().addAll(ivBG, btnStart, btnHelp, btnExit);
 
-        btnHelp.setOnAction(e ->
-            MainApplication.getStage().setScene(helpMenu())
-        );
+        return scene;
+    }
 
-        btnExit.setOnAction(e ->
-            MainApplication.exit()
-        );
+    @SuppressWarnings("Duplicates")
+    public static Scene helpMenu(Scene prevScene) {
 
-        title.setTranslateY(-200);
+        Parent root;
 
-        root.getChildren().addAll(title, btnStart, btnHelp, btnExit);
+        try {
+            root = FXMLLoader.load(Menu.class.getResource("/assets/ui/menus/help_menu.fxml"));
+            Controller.prevScene = prevScene;
+            Scene scene = new Scene(root, MainApplication.WINDOW_SIZE_X, MainApplication.WINDOW_SIZE_Y);
+            scene.getStylesheets().add("/assets/ui/stylesheets/style.css");
 
-        return scene;
+            return scene;
```

Comment:
This is just me being a low level assembly programmer, but typically it is better to have only one exit from a function where possible. in this case you can fix up this method by first creating `Scene result = null` at the beginning of the method. Then in the try statement set `result = scene`. This way you only have one return. It's also typically a bad idea to return within a try catch block as it could skip a cleanup step in a finally block if present. (This applies to all following methods)

Reply:
Really the only time you should be returning in a method is at the end, or if a method is called with parameters that don't fulfill the requirements of the operations that your method is performing.